GMPWrapper/CMakeLists.txt

```diff
@@ -25,6 +25,8 @@ gaudi_subdir(GMPWrapper v1r0)
 # APS not sure when that future is
 gaudi_depends_on_subdirs(GaudiAlg GaudiKernel)
 gaudi_install_python_modules()
+gaudi_install_scripts()
+install(DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/examples DESTINATION ${CMAKE_PROJECT_NAME})
```

Comment:
just GMPWrapper or share/GMPWrapper a.k.a. `${CMAKE_INSTALL_DATADIR}/${CMAKE_PROJECT_NAME}`?

Reply:
@fdplacido you removed `CMAKE_INSTALL_DATADIR` in the last commit - was that on purpose?

Reply:
Yes sorry, this was failing to install in my tests, should this go back?